setup.py

```diff
@@ -65,7 +67,8 @@ def add_version_constraint_or_raise(current_line, current_requirements, add_if_n
                 if is_requirement(line):
                     add_version_constraint_or_raise(line, requirements, True)
                 if line and line.startswith('-c') and not line.startswith('-c http'):
-                    constraint_files.add(os.path.dirname(path) + '/' + line.split('#')[0].replace('-c', '').strip())
+                    constraint_files.add(os.path.dirname(
```

Comment:
If this was updated because of line length, then following format would be more readable. If not then, this change isn't necessary.
```
constraint_files.add(
    os.path.dirname(path) + '/' + line.split('#')[0].replace('-c', '').strip()
)
```